packages/fxa-auth-server/lib/senders/emails/renderer.ts

```diff
@@ -0,0 +1,22 @@
+import ejs = require('ejs');
+import fs = require('fs');
+import path = require('path');
+import prettier = require('prettier');
+import mjml2html = require('mjml');
+import * as templates from './templates';
+
+const context = {
+  message: 'Hello Yogita',
+  buttonText: 'Sync device',
+  onDesktopOrTabletDevice: true,
+};
+
+let templateName: keyof typeof templates;
+for (templateName in templates) {
+  const template = ejs.compile(templates[templateName].render());
+  const mjmlTemplate = template(context);
+  const htmlTemplate = mjml2html(mjmlTemplate).html;
+  let prettyHtml = prettier.format(htmlTemplate, { parser: 'html' });
+  let outputFile = path.join(__dirname, 'dist', `${templateName}.html`);
+  fs.writeFileSync(outputFile, prettyHtml);
```

Comment:
You probably know this already but we probably won't need this `writeFileSync` when Storybook is integrated. Fine for now though. :)

Reply:
Since Storybook is the very next step I'm tempted to suggest that we try and convert this over in this PR. Could look something like this:

```ts
export function render(
  templateName: keyof typeof templates,
  context: Record<any, any>
) {
  const template = ejs.compile(templates[templateName].render());
  const mjmlTemplate = template(context);
  return mjml2html(mjmlTemplate).html;
}
```

Then again, this may look entirely different than what Les's Storybook implementation had in mind.